spec.html

```diff
@@ -185,7 +189,7 @@ <h2 class="nojs">Specification Version 1.1</h2>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x11" e_name int64</td>
+                    <td class="fix ex">"\x11" e_name uint64</td>
```

Comment:
It's documented on the same page: https://github.com/mongodb/bsonspec.org/pull/46/files/e611fcd478fd2fccba84628f4bc16e8036ddb03c#diff-3540caefa502006d8a33cb1385720803R300